plugins/GSdx/Renderers/Vulkan/GSTextureVK.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+ *	Copyright (C) 2019 Hamish Elliott
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with GNU Make; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "stdafx.h"
+#include "GSTextureVK.h"
+
+GSTextureVK::GSTextureVK()
+{
+	memset(&m_desc, 0, sizeof(m_desc));
```

Comment:
use
```cpp
GSTextureVK::GSTextureVK()
	: m_desc({})
{
}
```

instead of ugly memset (know it's elsewhere but let's modernize while we work)